dist/dattobd.spec

```diff
@@ -7,7 +7,15 @@
 %global _kmod_src_root %{_usrsrc}/%{name}-%{version}
 
 # Location for systemd shutdown script
-%global _systemd_shutdown /lib/systemd/system-shutdown
+# "%{_vendor}" == "redhat" covers rhel, centos and fedora 
+# Ubuntu18 doesn't have /usr/lib/systemd/, but Ubuntu20 has both locations with the same content
```

Comment:
I don't get it, from code it looks like /usr/lib/systemd/ is present for redhat, but in comments is that it is present for ubuntu20

Reply:
/usr/lib/systemd is on all redhats no wmatter what version and on Ubuntu 20.  
But on Ubuntu 18 there is only /lib/systemd. 
I guess that for backward compatibility reasons Ubuntu 20 has both paths and the same contents in both of them.
I'll try to rephrase the comment.